src/components/UnpublishedListing/UnpublishedListingCardMeta.css

```diff
@@ -0,0 +1,23 @@
+.cardMeta {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  height: 34px;
+  padding: 0 16px;
+  margin-bottom: -6px;
+
+  font-size: .75em;
+  text-transform: uppercase;
+
+  background: #eee;
+}
+
+.meta {}
+.label {
+  padding: 5px .75em 4px .75em;
```

Comment:
Just use pixels for everything, browsers now handle them pretty well across display densities.